library/general/src/lib/installation/proposal_client.rb

```diff
@@ -270,5 +270,9 @@ def ask_user(_attrs)
     def description
       raise NotImplementedError, "Calling abstract method 'description'"
     end
+
+    def write
+      # doing nothing is OK. or is it? clarify the API, the docs, actual usage!
+    end
```

Comment:
so possible to overlook bugs. Actual usage is without write, only in very specific modules like used to be network or cio proposal use it as it keep value, so finish client call proposal client with write.